target/linux/kirkwood/patches-5.10/113-lenovo-ix2.patch

```diff
@@ -0,0 +1,708 @@
+--- a/arch/arm/boot/dts/kirkwood-lenovo_ix2.dts
++++ b/arch/arm/boot/dts/kirkwood-lenovo_ix2.dts
```

Comment:
Please change patch to regular file and move it to: target/linux/kirkwood/files/arch/arm/boot/dts/

Reply:
Done, new DTS created